sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1089,6 +1089,22 @@ def _lambda_handler(blocked_resources: Set['resources_lib.Resources'],
             FailoverCloudErrorHandlerV2._default_handler(
                 blocked_resources, launchable_resources, region, zones, error)
 
+    @staticmethod
+    def _aws_handler(blocked_resources: Set['resources_lib.Resources'],
+                     launchable_resources: 'resources_lib.Resources',
+                     region: 'clouds.Region',
+                     zones: Optional[List['clouds.Zone']],
+                     error: Exception) -> None:
+        output = str(error)
+        logger.info(f'AWS handler error: {output}')
+        # Block AWS if the credential has expired.
+        if output.find('InvalidCloudCredentials') != -1:
```

Comment:
```suggestion
        logger.info(f'AWS handler error: {error}')
        # Block AWS if the credential has expired.
        if isinstance(error, exceptions.InvalidCloudCredentials):
```
But I'm still not sure we need this handler at all.

Reply:
Updated and replied here for why we need this handler https://github.com/skypilot-org/skypilot/pull/5015#discussion_r2013289389